docs/Earn/liquid-staking/dot/dot-liquid-staking-mechanics.md

```diff
@@ -0,0 +1,183 @@
+
+* aDOTb ratio changes, thus all aDOTb balances “automatically” grow.
+
+Staking is well-described by its requirement, fees, rewards, validators, as well as smart contracts and workflow that shows the API endpoints to call to interact with these smart contracts.
+
+## Requirements
+
+The minimum requirements when staking are:
+
+* Minimum value to stake — 1 DOT.
+
+* Maximum value to stake — unlimited, at the user’s discretion.
+
+Staking can be repeated anytime, without waiting for the current staking to completed as staking on Polkadot is done by the ANKR-owned wallet where users send their stake directly.
+
+The minimum requirements when unstaking are:
+
+* Minimum value to unstake — 0 DOT.
+
+* Maximum value to unstake — staked amount + % of the reward, up to the user’s balance.
+
+* Release period — up to 7 days.
+
+## Fees
+
+When staking, the user pays:
+
+* 2% commission on the reward, to ANKR.
+
+When unstaking, the user pays no fees.
+
+The users must also count in the cost for outgoing transactions.
+
+## Rewards
+
+Validators/delegators receive rewards daily.
+
+ANKR backend calculates the ratio of rewards to the total stake amount weekly and publishes it to the aDOTb smart contract respectively.
+
+## Validators
+
+ANKR delegates the staked funds to trusted validators from Polkadot, which may change in 1 day or longer.
+
+## Smart contract
+
+Smart contracts addresses involved in DOT staking are:
+
+Polkadot deposit address:
+
+[ANKR deposit address](https://polkadot.subscan.io/account/11XAoN3fgVLJUmK4Ty2q2H5dGYYzRZe82F247SyzfwZbHpV) — address that receives the staked DOTs on Polkadot.
+
+Ethereum:
+
+[aDOTb](https://etherscan.io/address/0x5cc56c266143f29a5054b9ae07f3ac3513a7965e) — contract on the Ethereum network that mints or burns aDOTb tokens for the user in 1:1 ratio with the staked amount.
+
+[PolkadotPool](https://etherscan.io/address/0x59f767EC659E9FE01ebCf930465E2aD4Cc0F208e) — contract on the Ethereum network where the user sends their request to mint or burn aDOTb.
+
+## Staking Flow
+
+The flow consists of two parts: staking DOT and claiming aDOTb.
+
+### Staking DOT
+
+The workflow is both user- and ANKR-driven. The user part is Step 1 through 3, while the ANKR part is Step 3 through 5. User interacts with ANKR and Polkadot via the ANKR frontend. 
+
+Due to Polkadot backend architecture, we currently do not disclose our API endpoints to avoid scams or unregulated heavy traffic to the ANKR backend. 
+
+1. User requests the ANKR deposit address via the ANKR backend.
+
+2. User sends DOTs to the obtained ANKR deposit address via the ANKR backend.
+
+3. User sends their extrinsic id to the ANKR backend.
+
+4. Once the transaction has been finalized, the user can see their DOT balance on the StakeFi page the stake was made from.
+
+5. ANKR backend monitor deposits to the ANKR deposit address. Once new DOTs are available for staking, the backend:
+   1. Transfer the DOTs to one of our treasury accounts if necessary. 
```

Comment:
Please remove mention of our treasury accounts as this suggests that we are providing custody. This can lend us in trouble with SEC legislation.